extension/scripts/global/globalData.js

```diff
@@ -403,6 +403,7 @@ const DEFAULT_STORAGE = {
 				stackingMode: new DefaultSetting({ type: "boolean", defaultValue: false }),
 			},
 			profile: {
+				avgpersonalstats: new DefaultSetting({type: "boolean", defaultValue: false}),

```

Comment:
Why are we making it disabled by default ? It uses data on the page and does not require any user action. We can make it enabled by default.

Reply:
I don't agree with this, this is a fairly niche feature. The majority of our users don't need it, and won't understand what that number is doing there. I would keep it disabled by default.

Reply:
I agree with DeKleineKobini too now. We can keep this disabled by default.